styles/meals.css

```diff
@@ -0,0 +1,105 @@
+/* Recipe containers */
+
+.weekly-meals-section {
+    justify-content: center;
+    /* background-color: #DCA47C; */
+    height: 350px;
+    margin: 50px 30px;
+}
+
+.favourite-meals-section {
+    /* background-color: #DCA47C; */
+    height: 350px;
+    margin: 50px 30px;
+}
+
+
+.more-meals-section {
+    /* background-color: #DCA47C; */
+    height: 660px;
+    margin: 50px 30px;
+}
+
+/* Recipe blocks */
+.weekly-meals-content, .favourite-meals-content, .more-meals-content, .new-recipe-content {
```

Comment:
all of them has -content in the end.

So you can call this just a content.
Classes shouldn't be uniq across the page.

Reply:
Done